x-pack/plugins/security_solution/public/common/components/alerts_treemap/lib/chart_palette/index.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { clamp } from 'lodash/fp';
+
+import {
+  RISK_COLOR_LOW,
+  RISK_COLOR_MEDIUM,
+  RISK_COLOR_HIGH,
+  RISK_COLOR_CRITICAL,
+  RISK_SCORE_MEDIUM,
+  RISK_SCORE_HIGH,
+  RISK_SCORE_CRITICAL,
+} from '../../../../../detections/components/rules/step_about_rule/data';
+
+/**
+ * The detection engine creates risk scores in the range 1 - 100.
+ * These steps also include a score of "zero", to enable lookups
+ * via an array index.
+ */
+export const RISK_SCORE_STEPS = 101;
+
+/**
+ * Returns a color palette that maps a risk score to the risk score colors
+ * defined by the Security Solution.
+ *
+ * The pallet defines values for a risk score between 0 and 100 (inclusive),
+ * but in practice, the detection engine only generates scores between 1-100.
+ *
+ * This pallet has the same type as `EuiPalette`, which is not exported by
+ * EUI at the time of this writing.
```

Comment:
Do you think it's worth creating an issue in the EUI project to get the palette exported in the near future?

Reply:
> Do you think it's worth creating an issue in the EUI project to get the palette exported in the near future?

I reached out to the EUI team with some details, and will create an issue pending the outcome of that conversation.